resources/platform/linux/peach/pits/README.md

```diff
@@ -0,0 +1,75 @@
+Adding a pit:
+
+STEP 1:
+Make sure that your Peach pit is not incompatible. There are a few known
+problems with pits that are not solved and will not work. The known problems
+right now are:
+
+    1) Pits that rely on of Fixups do not work.
+    2) Pits that have a Padding field do not work EX: <Padding alignment="16" />
+    3) Pits with Numbers that do not have size equal to 8, 16, 24, 32 or 64 do 
+    not work
+
+STEP 2:
+In order to add a pit to clusterfuzz make sure the name of the file matches
+with the name of the main Data Model. For example PDF.xml has the name of the 
+data model as PDF.
+There are many places this could be, but it is usually easy to spot as it will
+be named something similar such as "PDFFileFormat" or "Pdf".
+In order to maintain convention make the title all caps. 
+
+STEP 3:
+Make sure the Pit is using the right type for numbers. Some Peach pits use hex 
+values for their numbers without explicitly stating it is a hex. This will cause
+the following error: `Peach.Engine.common.PeachException: Error: The default 
+value for <Number> elements must be an integer.` To explicitly state it as a hex 
+add the valueType field like so: 
+
+```<Number name="Marker1" valueType="hex" value="FF E1" size="16" token="true"/>```
```

Comment:
I think triple backticks should be on a separate line, otherwise it might not render correctly (github code review seems to render it wrong, for instance)

Reply:
Done